azurefunctions/src/main/java/com/microsoft/durabletask/azurefunctions/internal/middleware/OrchestrationMiddleware.java

```diff
@@ -39,12 +49,29 @@ public void invoke(MiddlewareContext context, MiddlewareChain chain) throws Exce
                 // The OrchestratorBlockedEvent will be wrapped into InvocationTargetException by using reflection to
                 // invoke method. Thus get the cause to check if it's OrchestratorBlockedEvent.
                 Throwable cause = e.getCause();
-                if (cause instanceof OrchestratorBlockedException){
+                if (cause instanceof OrchestratorBlockedException) {
                     throw (OrchestratorBlockedException) cause;
                 }
                 throw new RuntimeException("Unexpected failure in the task execution", e);
             }
-        });
+        }, dataConverter);

```

Comment:
To make it clear this is a member variable and not a local variable:
```suggestion
        }, this.dataConverter);

```